gen/bind_generator.jl

```diff
@@ -114,6 +98,11 @@ macro bind(sig::Expr, err::Union{String,Expr,Nothing} = nothing)
     else
         # turn e.g. h5f_close into H5Fclose
         cfuncname = Symbol(uppercase(prefix), rest)
+        # Remove the version number if present
+        if occursin(r"\d$", String(jlfuncname))
```

Comment:
what if we change this to
```
occursin(r"\d(?<!hdf5)$", String(jlfuncname))  #(negative lookbehind to exclude hdf5 at the end)
```
or 
```
occursin(r"\d$", String(jlfuncname)) && !occursin(r"hdf5$", String(jlfuncname))
```

so we can interely get rid of the exceptions list?